api/proto/info/info.proto

```diff
@@ -241,6 +241,16 @@ message ZInfoAppInstance {
   string uuid = 1;
   string name = 2;
   string domainName = 3;
+  string taskid = 4;
+}
+
+// ZInfoDeviceTasks - send a summary of tasks so that controller knows
+// how many app instance tasks are still present on the device. Controller
+// can then use it to wait for an app to be fully deleted etc.
+message ZInfoDeviceTasks {
+  string id = 1;
+  string name = 2;
+  string status = 3;
```

Comment:
Are there different kinds of tasks? Or is there only one? Just wondering if there is a need for "type" field.

Reply:
>Are there different kinds of tasks?

No